openreview/venue_request/process/revisionProcess.py

```diff
@@ -41,6 +41,22 @@ def process(client, note, invitation):
                     paper_withdraw_super_invitation.expdate = openreview.tools.datetime_millis(withdraw_submission_expiration)
                     client.post_invitation(paper_withdraw_super_invitation)
 
+            if len(conference.reviewer_roles) > 1:
```

Comment:
What about if multiple area chair or senior area chair roles are added (but reviewer roles is not modified)?